rllib/algorithms/ppo/tests/test_ppo_learner.py

```diff
@@ -110,7 +111,7 @@ def test_loss(self):
             learner_group.set_weights(trainer.get_weights())
             results = learner_group.update(train_batch.as_multi_agent())
 
-            learner_group_loss = results["loss"]["total_loss"]
+            learner_group_loss = results[ALL_MODULES]["total_loss"]
```

Comment:
btw, should this be results[ALL_MODULE]["loss"]["total_loss"] instead?

Reply:
no loss key is removed. We used to have loss key as a hierarchy which was solely introduced in the Learner API which was inconsistent with the old API anyways.